web/templates/dropdown_list.hbs

```diff
@@ -5,12 +5,14 @@
             <span class="dropdown-list-item-name">
                 {{#if bold_current_selection}}
                     <span class="dropdown-list-bold-selected">{{name}}</span>
-                    {{#if has_edit_icon}}
-                        {{> components/icon_button custom_classes="dropdown-list-edit dropdown-list-control-button" intent="neutral" icon="edit" aria-label=edit_icon_label }}
-                    {{/if}}
-                    {{#if has_delete_icon}}
-                        {{> components/icon_button custom_classes="dropdown-list-delete dropdown-list-control-button" intent="danger" icon="trash" aria-label=delete_icon_label }}
-                    {{/if}}
+                    <span class="dropdown-list-buttons">
+                        {{#if has_edit_icon}}
+                            {{> components/icon_button custom_classes="dropdown-list-edit dropdown-list-control-button" intent="neutral" icon="edit" aria-label=edit_icon_label }}
+                        {{/if}}
+                        {{#if has_delete_icon}}
+                            {{> components/icon_button custom_classes="dropdown-list-delete dropdown-list-control-button" intent="danger" icon="trash" aria-label=delete_icon_label }}
+                        {{/if}}
+                    </span>
                 {{else}}
                     {{name}}
```

Comment:
We should add another span + class to wrap all of the otherwise bare text-nodes introduced by `{{name}}` throughout this template, something parallel to `.dropdown-list-bold-selected`--maybe `.dropdown-list-text-neutral` or something like that? That will aid in grid placement, and give us better control over alignment, ellipses, etc.